pshtt/utils.py

```diff
@@ -86,3 +88,11 @@ def format_domains(domains):
         formatted_domains.append(re.sub("^(https?://)?(www\.)?", "", domain))
 
     return formatted_domains
+
+
+def debug(message, divider=True):
+    if divider:
+        logging.debug("\n-------------------------\n")
+
+    if message:
+        logging.debug("%s\n" % message)
```

Comment:
Sorry to be a nit, but looking at the usage in `pshtt.py`, it looks like `divider=False` would be a better default and less surprising. That way `utils.debug("Message")` just outputs a message and doesn't cause an additional line to appear.

Also, when referencing this in `pshtt.py`, it would be more explicit and readable to reference the named argument directly. So for example (assuming a reversed default), a call might look like this:

```python
utils.debug("My message!", divider=True)
```

And to do it with no divider:

```python
utils.debug("My message!")
```